### PR TITLE
fix fov computation from fbxsdk

### PR DESCRIPTION
**Pull request by EscapeZero**
Need to do some fun checks of fbx internal camera model before computing FOV.

Useful FBXSDK example if we want to support more camera info later.
https://help.autodesk.com/view/FBX/2017/ENU/?guid=__cpp_ref__view_scene_2_set_camera_8cxx_example_html

### CODE REVIEW

File: src/fbx/Fbx2Raw.cpp [hunk 1]
```diff
@@ -367,13 +367,60 @@ static void ReadMesh(RawModel &raw, FbxScene *pScene, FbxNode *pNode, const std:
...
+            fovy = pCamera->FieldOfViewY;
+            fovx = VFOV2HFOV(fovy, 1.0 / apertureRatio);
+            break;
+        }
+        case FbxCamera::EApertureMode::eFocalLength: {
+            fovx = pCamera->ComputeFieldOfView(pCamera->FocalLength);
+            fovy = HFOV2VFOV(fovx, apertureRatio);
+            break;
+        }
+    }
```
Comment (AlexanderSjoholm) [inline src/fbx/Fbx2Raw.cpp]: Would perhaps make sense to add a default case where you set the fovx/y to pCamera->FieldOfViewX/Y. Optionally I guess there should be a default case where it throws or similar. Carrying on with fov = 0 seems bad.

Comment (EscapeZero) [reply src/fbx/Fbx2Raw.cpp]: The switch covers all possible EApertureMode values in current sdk; we could add some default case to try and future proof it, but pCamera->FieldOfViewX/Y and FocalLength have non-zero default values in that case. So then you get weird values in the resulting glTF file and it is hard to figure out where things went wrong (this is how master behaves currently).

Would you prefer default case now knowing it is exhaustive switch? I don't mind either way, so your call.

Comment (AlexanderSjoholm) [reply src/fbx/Fbx2Raw.cpp]: I see. In that case, just to make it easier to debug future issues, we could log a warning/error in the default case
fmt::printf("Warning:: Unsupported ApertureMode");

How about that?

**Comment by zellski** [comment]
Nice job, thanks guys!

**Pull request merged**